README.md

```diff
@@ -27,10 +51,39 @@ Next set up the development and test databases. Make sure `npx` is working first
 
 - `./script/init_db`
 
-If these command fail due to `could not connect to server: No such file or directory`, check the log file at `/usr/local/var/log/postgres.log`. If you see an error that looks similar to `The data directory was initialized by PostgreSQL version 11, which is not compatible with this version 13.2`, you have a previous version of postgresql installed that is conflicting with this one. The easiest way to resolve this is to `mv` the old data directory and run `initdb` with the new version of postgres. See https://gist.github.com/olivierlacan/e1bf5c34bc9f82e06bc0 for more details.
-
```

Comment:
Removed this lengthy discussion of an edge-case setup gotcha in favor of a sentence below asking for people to file an issue if they run into any problems.